### PR TITLE
AppRoles_default tree

### PR DESCRIPTION
**Pull request by dBianchii**

### CODE REVIEW

**Comment by dBianchii** [comment]
Current dependencies on/for this PR:
* **#143** <a href="https://app.graphite.dev/github/pr/dBianchii/kodix-turbo/143?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dBianchii/kodix-turbo/143?utm_source=stack-comment).

File: packages/db/src/seed/seed.ts [hunk 1]
```diff
@@ -131,36 +61,72 @@ async function main() {
...
+        );
+      }
+    }
+  }
+  console.log("🌱 - ✅ Seed input validation passed!");
+}
+
 main()
   .then(() => {
     console.log("🌳 Fully seeded!");
```
Comment (wcbianchi) [inline packages/db/src/seed/seed.ts:132]: <i class='graphite__hidden'>[Re: line 133]<br/><br/></i>

Show<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dBianchii/kodix-turbo/143?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by wcbianchi — APPROVED** [review]
Ótimo filho, mas eu não vi o objeto de mapeamento das definições de permissão

**Comment by dBianchii** [comment]
rewritten and applied here https://github.com/dBianchii/kodix-turbo/pull/146

**Pull request closed without merging**